rust/abacus-core/src/types/interchain_gas.rs

```diff
@@ -0,0 +1,9 @@
+use ethers::types::U256;
```

Comment:
Yk, there's nothing wrong with having serval structs int the same file. Since this seems pretty small I wonder if we really need a module for it.